packages/docs/docs/.vuepress/config.js

```diff
@@ -42,6 +42,13 @@ module.exports = ctx => ({
         selectText: 'Languages',
         ariaLabel: 'Select language',
         editLinkText: 'Edit this page on GitHub',
+        notFoundLinkText: 'Take me home.',
```

Comment:
I don't think we need to set this for the `/` locale as these are the default values right?